test/test_ops.py

```diff
@@ -1,5 +1,5 @@
 # Owner(s): ["module: unknown"]
-
+# ruff: noqa: F841
```

Comment:
This file is quite critical, we should re-enable this one cc @jbschlosser maybe :D

Reply:
No need to worry, this file has very few errors, I'm fixing them.